### DISCUSSION TITLE
Welcome to react-native-gifted-charts Discussions!

### DISCUSSION THREAD

**Discussion by Abhinandan-Kushwaha** [Abhinandan-Kushwaha/react-native-gifted-charts#217] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kayGweb** [comment]
Hello guys,
First, I enjoy my work with the library, which provides most of my current client's needs.  However, I do have a question concerning the limitations or capabilities in its current form.  We have a requirement that one of the lines in a line chart has an areaChart feel with gradient and opacities, and the other lines do not have the areaChart staying.  Is that possible??  If so, how would I execute that?

Thanks,

**Reply by Abhinandan-Kushwaha** [reply]
Hi @kayGweb 👋
This is now possible in the latest version of the library. You can give one of the lines an areachart feel using the `areaChart1` `areaChart2` ... `areaChart5` props.

**Comment by sanjay186** [comment]
HI 

Need expo web support. its working perfect in mobile.
it is breaking when i use it in expo web.
